vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt

```diff
@@ -68,6 +70,15 @@ class CreateRoomViewModel @AssistedInject constructor(@Assisted private val init
         }
     }
 
+    private fun initUserDomain() {
+        val displayName = session.run { getUser(myUserId) }
+                ?.displayName
+                ?.takeUnless { it.isEmpty() }
+                ?: TchapUtils.computeDisplayNameFromUserId(session.myUserId).orEmpty()
```

Comment:
please add a todo : `// TODO: it should be better to update User.getBestName function to compute the displayname`